R-notebooks/safegraph_dashboard.Rmd

```diff
@@ -1,67 +1,50 @@
 ---
 title: "Safegraph dashboard"
 author: "Delphi Lab"
-date: "`r format(Sys.time(), '%B %d, %Y')`"
+date: "`r format(Sys.time(), '%B %d, %Y - %H:%M:%S')`"
 output:
   html_document:
     code_folding: hide
+params:
+  source: "safegraph"
+  signal: "bars_visit_num"
```

Comment:
`bars_visit_num` is likely to have different behavior from `full_time_work_prop`, since they come from different data pipelines. Is there a way we can make these dashboards support multiple signals? Otherwise we'll want to split up sources that combine multiple pipelines so that we have one dash per pipeline.

Reply:
Agreed. Reverted back to median_home_dwell_time for now. I'll add the new signals in a separate PR.